safe_transaction_service/history/indexers/ethereum_indexer.py

```diff
@@ -164,6 +164,11 @@ def get_to_block_number(
         :param current_block_number:
         :return: Top block number to process
         """
+        if self.block_process_limit == 1:
```

Comment:
I would propose removing this and just:

```
 return min(
            from_block_number + self.block_process_limit - 1,
            current_block_number - self.confirmations,
)
```

Reply:
Oh i get it, then we will never index the last block. Could you add a test for this so we don't break it in the future?